packages/contracts-bedrock/.gas-snapshot

```diff
@@ -17,7 +17,7 @@ AttestationStationTest:test_attest_bulk_succeeds() (gas: 544387)
 AttestationStationTest:test_attest_individual_succeeds() (gas: 472850)
 AttestationStationTest:test_attest_single_succeeds() (gas: 491962)
 BlockOracle_Test:test_checkpointAndLoad_succeeds() (gas: 58396)
-BlockOracle_Test:test_load_noBlockHash_reverts() (gas: 12827)
+BlockOracle_Test:test_load_noBlockHash_reverts() (gas: 13681)
```

Comment:
What's with these changing values? How did you generate them? I've not had to modify these before, so might be worth adding to the doc?

Reply:
I don't know why these values change, but we can use `pnpm gas-snapshot` to generate it. However, the result on my MAC is different from we have from CI. I have to manually overwrite some gas usage. According to this https://github.com/foundry-rs/foundry/issues/4917 , the seed might be different.

Reply:
BTW, these changes are from the upstream.